database/migrations/000001_init.up.sql

```diff
@@ -281,6 +281,7 @@ CREATE INDEX idx_roles_project_id ON roles(project_id);
 CREATE UNIQUE INDEX roles_organization_id_name_lower_idx ON roles (organization_id, LOWER(name));
 CREATE INDEX idx_provider_access_tokens_project_id ON provider_access_tokens(project_id);
 CREATE UNIQUE INDEX repositories_repo_id_idx ON repositories(repo_id);
+CREATE UNIQUE INDEX repositories_cursor_pagination_idx ON repositories(project_id, provider, repo_id);
```

Comment:
(I haven't really read the PR in full, just replying to this one bit for now..)

The reason the tests are failing is that whenever you change the DB schema you need to add two new migration files, not modify the old ones. Currently the largest one is 11, so you'd add 12. Also, one thing that bit me in the beginning is that you need to add both up and down migrations (even if the down was empty).

Reply:
I see. Added new migration files.